dependencies.yaml

```diff
@@ -101,10 +132,8 @@ dependencies:
       - output_types: [conda, pyproject, requirements]
         packages:
           - cython>=3.0.3
-          - *rmm_unsuffixed
-          - pylibcudf==25.6.*,>=0.0.0a0
```

Comment:
We definitely need pylibcudf to build the cython.

Reply:
Yep, it's just been moved to https://github.com/rapidsai/rapidsmpf/pull/269/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6R280